cli-output/src/cli_output.rs

```diff
@@ -582,6 +582,41 @@ pub struct CliEpochReward {
     pub apr: f64,
 }
 
+fn show_votes_and_credits(
+    f: &mut fmt::Formatter,
+    votes: &[CliLockout],
+    epoch_voting_history: &[CliEpochVotingHistory],
+) -> fmt::Result {
+    if votes.is_empty() {
+        return Ok(());
+    }
+
+    writeln!(f, "Recent Votes:")?;
+    for vote in votes {
+        writeln!(f, "- slot: {}", vote.slot)?;
+        writeln!(f, "  confirmation count: {}", vote.confirmation_count)?;
+    }
+    writeln!(f, "Epoch Voting History (without skipped slots):")?;
```

Comment:
Actually, I wanted to show skipped slots but it's not easy task. So, at least show a note about it to educate/inform users correctly.

Reply:
tried to be shorter but this might be too ambuguious. `(skipped slots not considered)`/`(missed credits can be skipped slots)` might be better?

Reply:
I do find that language a little confusing.
Maybe something like this on a new line? `* not all epoch slots may have been available to vote on`

Reply:
@CriesofCarrots Thanks for suggestion! I've further adjusted the wording based on yours to make interested users be able to have a clue to know more: https://github.com/solana-labs/solana/pull/13374/commits/14bdcba429c22289366494ea276560be967c8b9b How does this look?

Reply:
Much better. Thanks for the polish!